tasks/sync_repos.py

```diff
@@ -161,12 +161,15 @@ async def sync_repos_affected_repos_known(
                 missing_repo_service_ids=missing_repo_service_ids,
                 num_missing_repos=len(missing_repo_service_ids),
                 existing_repos=existing_repos,
+                repository_service_ids=repository_service_ids,
             ),
         )
 
         # Get info from provider on the repos we don't have
         repos_to_search = [
-            x[1] for x in repository_service_ids if x[0] in missing_repo_service_ids
+            x[1]
+            for x in repository_service_ids
+            if str(x[0]) in missing_repo_service_ids
```

Comment:
i don't see where you are changing the type of this input in this PR. was this broken the whole time?

Reply:
I believe it was broken this whole time, yes.
I know, my bad 😓